_themes/sass/_theme_rst.sass

```diff
@@ -174,19 +173,18 @@
 
 
   // This is the #href that shows up on hover. Sphinx's is terrible so I hack it away.
-  h1, h2, h3, h4, h5, h6, dl dt, p.caption
+  h1, h2, h3, h4, h5, h6, dl dt, p.caption, caption
```

Comment:
I was avoiding changing the existing in case they style different types of captions differently. Would need to do a full review before I wanted to delete it everywhere.